### PR TITLE
Add pathology and notes entries

### PR DESCRIPTION
**Pull request by valosekj**
This PR adds `pathology` and `notes` entries to the `participants.json` and `participants.tsv` files.

The `pathology` and `notes` entries are added based on the qualitative assessment of T2w images performed by two radiologists.

### CODE REVIEW

**Review by jcohenadad — APPROVED** [review]
Looks good, than you Jan!

**Comment by jcohenadad** [comment]
I let you merge when you want

**Review by sandrinebedard — APPROVED** [review]

File: participants.json [hunk 1]
```diff
@@ -50,5 +50,18 @@
...
         "Description": "Researcher"
-    }
+    },
+    "pathology": {
+        "Description": "The diagnosis of pathology of the participant",
+        "LongName": "Pathology name",
+        "Levels": {
+            "HC": "Healthy Control",
+            "DCM": "Degenerative Cervical Myelopathy (synonymous with CSM - Cervical Spondylotic Myelopathy)",
+            "MildCompression": "Asymptomatic cord compression, without myelopathy",
```
Comment (mguaypaq) [inline participants.json]:
```suggestion
            "MildCompression": "Asymptomatic cord compression, without myelopathy"
```

Comment (valosekj) [reply participants.json]: Thank you for this fix!

**Review by mguaypaq — APPROVED** [review]

**Pull request merged**